MonoidalCategories/doc/Doc.autodoc

```diff
@@ -1,2 +1,15 @@
-@Chapter Tensor Product and Internal Hom
+@Chapter Monoidal Categories
 
+@Section Monoidal Categories
+
+@Section Distributive Monoidal Categories
```

Comment:
Could you rename
`Distributive Monoidal Categories`
to
`Additive Monoidal Categories`?
This request also affects the file names:
DistributiveMonoidalCategories.gd,
DistributiveMonoidalCategories.gi,
DistributiveMonoidalCategoriesMethodRecord.gi.

Curious: there is already a file named `AdditiveMonoidalCategoriesDerivedMethods.gi`.

Reason for this request: so far, the distributivity primitives are only formulated for the compatibility of direct sums and tensorproducts, and thus make only sense in an additive context.

Reply:
I will.

The reason for not renaming them was probably to abuse the code in `DistributiveMonoidalCategories.gd` for the case when the product distributes over the coproduct (e.g. in distributive lattices) as the word `DirectSum` does not occur explicitly in this file. But I now see it is cleaner to follow your suggestion and reintroduce distributive/co-distributive (co)cartesian categories explicitly, maybe even in `Toposes`.

Reply:
```suggestion
@Section Additive Monoidal Categories
```